components/cpp/prophet-wasmstan/wit/prophet-wasmstan.wit

```diff
@@ -166,8 +168,8 @@ interface types {
 }
 
 interface optimizer {
-    use types.{inits, data, optimize-opts, optimize-output};
-    optimize: func(init: inits, data: data, opts: optimize-opts) -> result<optimize-output, string>;
+    use types.{inits, optimize-opts, optimize-output};
+    optimize: func(init: inits, data: string, opts: optimize-opts) -> result<optimize-output, string>;
 }
```

Comment:
TODO: add comments to this interface, function, and args. Maybe use a newtype for `data`